atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -337,6 +338,14 @@ def get_build_image_override(workflow, fallback=NO_FALLBACK):
     return get_value(workflow, 'build_image_override', fallback)
 
 
+def get_default_image_build_method(workflow, logger, fallback=CONTAINER_DEFAULT_BUILD_METHOD):
+    value = get_value(workflow, 'default_image_build_method', fallback)
+    if value not in CONTAINER_BUILD_METHODS:
+        logger.error("invalid default_image_build_method '{}'; using default".format(value))
+        return fallback
```

Comment:
These two lines are not covered by unit testing.

Reply:
Should be now